app/views/play/CampaignView.js

```diff
@@ -1309,20 +1325,21 @@ class CampaignView extends RootView {
   }
 
   onMouseEnterPortals (e) {
-    if (!this.campaigns?.loaded || !this.sessions?.loaded) { return }
+    if (this.isCatalyst) return // Skip for catalyst view
     this.portalScrollInterval = setInterval(this.onMouseMovePortals, 100)
     return this.onMouseMovePortals(e)
   }
 
   onMouseLeavePortals (e) {
-    if (!this.portalScrollInterval) { return }
+    if (this.isCatalyst) return // Skip for catalyst view
```

Comment:
same. is protalScrollInterval not used anymore?

Reply:
Yep. screwed up by AI